src/option_types.cc

```diff
@@ -21,4 +21,21 @@ UnitTest test_option_parsing{[]{
     check(DebugFlags::Keys | DebugFlags::Hooks, {"hooks|keys"});
 }};
 
+bool option_add_from_strings(Vector<RangeAndString, MemoryDomain::Options>& opt, ConstArrayView<String> strs)
+{
+    auto vec = option_from_strings(Meta::Type<Vector<RangeAndString, MemoryDomain::Options>>{}, strs);
+    size_t original_size = opt.size();
+    opt.insert(opt.end(),
```

Comment:
This returns an iterator to the first inserted element, so no need for original_size